readme.md

```diff
@@ -357,7 +356,7 @@ GitHub implemented dozens of features that used to be part of Refined GitHub 
 - [Blog post](https://github.blog/changelog/2019-12-19-improved-attribution-when-squashing-commits/): [Adds `co-authored-by` to the commit when merging PRs with multiple committers.](https://user-images.githubusercontent.com/1402241/51468821-71a42100-1da2-11e9-86aa-fc2a6a29da84.png)
 - [Blog post](https://github.com/sindresorhus/refined-github/pull/3015): [Makes it easier to tell apart commits added to the current PR versus plain commits that reference the PR.](https://user-images.githubusercontent.com/1402241/64478939-398b0a80-d1da-11e9-8c6a-bb98668cb78c.gif)
 - [Blog post](https://github.blog/changelog/2020-06-23-design-updates-to-repositories-and-github-ui/): [Widens the discussion search box.](https://user-images.githubusercontent.com/1402241/55069759-bceaf080-50bf-11e9-84d0-7707de2eb9e9.png)
-
+- [Blog post](https://github.com/sindresorhus/refined-github/issues/3776): [Allows the <kbd>↑</kbd> and <kbd>↓</kbd> keys to cycle "popover lists" (labels, milestones, etc).](https://user-images.githubusercontent.com/37769974/59158786-6fd2c400-8add-11e9-9db1-db80186fa6ea.gif)
```

Comment:
“Blog post” is starting to not make sense. Either we drop the link or we change the wording. It’s not particularly useful, I mean the whole list isn’t super useful either. 

What if we replace it with a PR search link that points to every PR with “drop” in the title? Well also have to update some old PRs that don’t use the word.

Reply:
Agree. This PR or a lint?

Reply:
I was thinking the same. I was looking at the changes and when I say "blog post" I was confused.
What about simply using a label like `adopted by github`? So what you could use the link:
https://github.com/sindresorhus/refined-github/pulls?q=is%3Apr+is%3Aclosed+label%3A%22adopted+by+github%22

Reply:
> What about simply using a label

We could use a more generic `dropped feature` label, but we don't really need another label since the PR title should include that keyword already.

> Agree. This PR or a lint?

This PR can just avoid making changes to that section. In a lint you could drop the list and if you have time maybe also track down any PRs that don't have the "Drop" keyword in the title

Reply:
```suggestion
```